### PR TITLE
Remove parameters from `CustomMetadata` struct

### LINKED ISSUE #375: Change `CustomMetadata` for asset registry pallet
We should adjust the `CustomMetadata` struct to our needs. Since we don't really need any of the additional metadata that is defined at the moment, we can also leave it empty.

This is related to [this](https://github.com/pendulum-chain/pendulum/issues/275) ticket and required for [this](https://github.com/pendulum-chain/pendulum/pull/276) PR.

### PR DESCRIPTION
**Pull request by ebma**
This PR changes the `CustomMetadata` struct to something we can use with our pendulum runtimes. The `fee_per_second` parameter seems to be used for XCM calls, so it makes sense to keep it.

For the review, please consider [this](https://github.com/pendulum-chain/spacewalk/issues/375#issuecomment-1640554241) comment.

Closes #375.

### CODE REVIEW

**Comment by TorstenStueber** [comment]
@ebma where was the `CustomMetadata` struct actually used before? I don't find any references.

**Comment by TorstenStueber** [comment]
As [mentioned in the associated issue](https://github.com/pendulum-chain/spacewalk/issues/375#issuecomment-1645699523), I propose that we remove all entries from `CustomMetadata` for now.

File: primitives/src/lib.rs [hunk 1]
```diff
@@ -556,10 +556,12 @@ impl CurrencyId {
...
 }
 
+/// This struct defines the custom metadata for an asset registered on an asset registry pallet.
 #[derive(scale_info::TypeInfo, Encode, Decode, Clone, Eq, PartialEq, Debug)]
 pub struct CustomMetadata {
-	pub fee_per_second: u128,
-	pub coingecko_id: Vec<u8>,
+	/// The fee charged for every second that an XCM message takes to execute.
+	/// When `None`, the `default_per_second` will be used instead.
+	pub fee_per_second: Option<Balance>,
```
[the remark this answers is not in the document]
Comment (ebma) [reply primitives/src/lib.rs]: I changed it to a unit struct now. Or would you suggest otherwise @b-yap?

**Review by TorstenStueber — APPROVED** [review]

**Comment by ebma** [comment]
I'm merging this although the CI job is failing because the issue is related to #382 and not related to this PR.

**Pull request merged**